### ISSUE TITLE
Make double-click on whiteboard open textinput-field like on tldraw.com

### ISSUE THREAD

**Issue by Ithanil** [bigbluebutton/bigbluebutton#17889]
**Is your feature request related to a problem? Please describe.**
Creating many text fields on the whiteboard can be tedious. For every text field you have to re-select the tool and then move the cursor to the desired location again.

**Describe the solution you'd like**
A double click in the whiteboard area opens a text field at that location (like on tldraw.com).

**Comment by jorgelondo** [comment]
I totally agree, when selecting a tool it should stay selected, I think the proper behavior should be something like in design tools like Adobe illustrator.

I think that to achieve a more fluid operation, you should leave the tool selected and create a new text field by clicking outside the previously created text field, the same with circles, lines, squares, etc...

The goal is to be able to create several of the same elements without having to go back each time to select the same tool
